src/subcommand/server.rs

```diff
@@ -1493,6 +1494,36 @@ impl Server {
     })
   }
 
+  async fn content_delegator(
+    Extension(index): Extension<Arc<Index>>,
+    Extension(settings): Extension<Arc<Settings>>,
+    Extension(server_config): Extension<Arc<ServerConfig>>,
+    Path(inscription_id): Path<InscriptionId>,
+    accept_encoding: AcceptEncoding,
+  ) -> ServerResult {
+    task::block_in_place(|| {
+      if settings.is_hidden(inscription_id) {
+        return Ok(PreviewUnknownHtml.into_response());
+      }
+
+      let inscription = index
+        .get_inscription_by_id(inscription_id)?
+        .ok_or_not_found(|| format!("inscription {inscription_id}"))?;
+
+      if inscription.delegate().is_some() {
```

Comment:
Why check if there is a delegate? We could remove this check and unconditionally return the content of the inscription instead.

Reply:
So you mean return envelope regardless?

My thought was if you guys cache this on the CDN like `/content/` then it would be caching it twice? Why would we return redundant information if `/content/` already has it if no delegate is present?

Reply:
The CDN is free, so it's probably better to make the code and behavior simpler than try to save CDN resources.